keyboards/mechwild/sugarglider/sugarglider.h

```diff
@@ -0,0 +1,33 @@
+// Copyright 2023 Kyle McCreery
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include "quantum.h"
+
+typedef union {
+    uint32_t raw;
+    struct {
+        uint8_t dpi_config;
+        int16_t dt_term_config;
+    };
+} keyboard_config_t;
+
+extern keyboard_config_t keyboard_config;
+extern uint16_t          dpi_array[];
+
+enum keyboard_keycodes {
+#ifdef VIA_ENABLE
+    DPI_UP = USER00,
+#else
+    DPI_UP = SAFE_RANGE,
+#endif
```

Comment:
VIA is now using the QK_KB range for custom keycodes, so no need to set separate ranges. (at least when compiled from develop, which is due to merge in a few days)
```suggestion
    DPI_UP = QK_KB,
```